mitosheet/mitosheet/mito_backend.py

```diff
@@ -68,12 +69,18 @@ def __init__(
         # Setup the MitoConfig class
         self.mito_config = MitoConfig() # type: ignore
 
-        custom_sheet_functions_path = self.mito_config.get_mito_config()[MITO_CONFIG_CUSTOM_SHEET_FUNCTIONS_PATH]
+        # Set up the Mito Logger class
+        jupyter_log_server_url = self.mito_config.jupyter_log_server_url
+        if jupyter_log_server_url:
+            jupyter_log_server_batch_interval = self.mito_config.jupyter_log_server_batch_interval
+            self.mito_log_uploader = MitoLogUploader(jupyter_log_server_url, jupyter_log_server_batch_interval)
```

Comment:
This conditional setting up the uploader isn't great practice -- we should have an else case (setting it to None) -- so you can always access it